plugins/vue-darkmode.js

```diff
@@ -0,0 +1,24 @@
+import Vue from 'vue';
+
+Vue.use(function () {
+  const darkMode = {
+    state: Vue.observable({ isDark: false, isLight: true }),
```

Comment:
Why do we need to use two booleans here? Can't we use only one property like `{ theme: "dark" | "light" }` ?

Reply:
Thanks for the review 🙇🏻

Reply:
Sure, you're welcome @Marsup . Thanks for the fix. 🙏